setup.py

```diff
@@ -57,7 +57,7 @@ def run(self):
     def build_extension(self, ext):
         try:
             build_ext.build_extension(self, ext)
-        except (DistutilsExecError,
+        except (CCompilerError, DistutilsExecError,
```

Comment:
This is a workaround. We need a real fix, finding out why the file is missing or having a clear explanation of why this is happening.

Reply:
Okay, it might make sense. Let's wait for @asvetlov.
P.S. I still want to see this tested in CI.

Reply:
Nope. We already discussed the problem with @asvetlov in #3162 and I probably caught his idea and pain about development environment. Though the build is broken anyway: it is impossible to build/install from sources without gcc. So I plan to improve setup.py.

Reply:
Feel free to try, but I'll require adding env combos to Travis CI as mandatory anyway.

Reply:
And while you're on it you might want to extend docs to mention how things work.

Reply:
```I'll require adding env combos to Travis CI as mandatory anyway.

@asvetlov mentioned that this is the bad idea, because it will increase even more the testing time.

Reply:
Not really, just add it and then I'll make sure that it doesn't hit performance. I'll basically do to it what I did to OS X and it'll be a fine compromise: running those only on cron as opposed to having them in each build. Andrew trusts me to handle CI.

P.S. If we recognize that we need it on each commit we could always move it to another CI service.

Reply:
Remark: while working on changing `setup.py` it might be worth adding some env var for toggling strict mode for dev.

It could be smth like:
```python
import os
...
class ve_build_ext(build_ext):
    ...
    def build_extension(self, ext):
        valid_build_exceptions = {CCompilerError, DistutilsExecError, DistutilsPlatformError, ValueError}
        suppressed_build_exceptions = set()

        if os.getenv('AIOHTTP_DEV_MODE'):
            suppressed_build_exceptions = {CCompilerError, DistutilsExecError, DistutilsPlatformError}

        try:
            build_ext.build_extension(self, ext)
        except tuple(valid_build_exceptions - suppressed_build_exceptions) as build_exc:
            raise BuildFailed() from build_exc
```

cc: @asvetlov

Reply:
Yeah, that's what I planned to implement.